stable/kubernetes-dashboard/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```

Comment:
I would prefer to use `{{ template "name" . }}` instead of  `{{ template "fullname" . }}` for the `app` labels.